app.json

```diff
@@ -42,10 +42,6 @@
     "MAILGUN_KEY": {
       "description": "The token for authenticating against the Mailgun API"
     },
-    "MAILGUN_BATCH_CHUNK_SIZE": {
-      "description": "Maximum number of emails to send in a batch",
```

Comment:
why we removed this

Reply:
Mailgun's API only supports batching if the email contents are the same, but sent to many recipients. In the case of open-discussions we'll very likely never do this since the email content will be personalized per-recipient.